wgengine/userspace.go

```diff
@@ -841,6 +872,22 @@ func (e *userspaceEngine) Reconfig(cfg *wgcfg.Config, routerCfg *router.Config,
 		}
 	}
 
+	if isSubnetRouterChanged && e.birdClient != nil {
+		e.logf("wgengine: Reconfig: configuring BIRD")
+		var err error
+		if isSubnetRouter {
+			err = e.birdClient.EnableProtocol("tailscale")
+		} else {
+			err = e.birdClient.DisableProtocol("tailscale")
+		}
+		if err != nil {
+			// Log but don't fail here.
+			e.logf("wgengine: Reconfig: configuring BIRD failed: %v", err)
```

Comment:
I'm not 100% sure but I think we've been moving to getting the word "error" in all failure log prints to ease grepping logs for "error" to find bad stuff.

So like "error configuring BIRD: %v"